DiscordLabs.org/dist/presence.js

```diff
@@ -0,0 +1,64 @@
+var presence = new Presence({
+    clientId: "660894911331172372",
+    mediaKeys: false
+}), strings = presence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused"
+});
+var browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+    let presenceData = {
+        largeImageKey: "logo",
+        smallImageKey: "author",
+        smallImageText: "Developer: Dominik#7464"
```

Comment:
Please dont do this, you could use some of our small images https://github.com/PreMiD/Presences/tree/master/.resources

Reply:
smallImageText: "Developer: Dominik#7464"
This is also not allowed to do.